spec/rails_helper.rb

```diff
@@ -88,12 +88,8 @@
     DatabaseCleaner.strategy = :truncation
   end
 
-  config.before(:each, type: :controller) do
-    @request.host = 'localhost:3000'
-  end
-
   config.before(:each, type: :request) do
-    host!('localhost:3000')
+    host!(ENV.fetch('HOST', 'localhost') + ":8080")
```

Comment:
Did you mean to include this as part of the commit?

Reply:
I didn't. I'll revert.

Reply:
We have a static host again